src/Http/Http.Extensions/gen/StaticRouteHandlerModel/EndpointParameter.cs

```diff
@@ -57,6 +60,38 @@ public EndpointParameter(IParameterSymbol parameter, WellKnownTypes wellKnownTyp
         }
     }
 
+    private static bool TryGetParsability(IParameterSymbol parameter, WellKnownTypes wellKnownTypes, [NotNullWhen(true)]out Func<string, string, string>? parsingBlockEmitter)
+    {
+        if (ParsabilityHelper.GetParsability(parameter.Type, wellKnownTypes, out var parsabilityMethod) == Parsability.NotParsable)
+        {
+            parsingBlockEmitter = null;
+            return false;
+        }
+
+        Func<string, string, string>? preferredTryParseInvocation = parsabilityMethod switch
```

Comment:
Nit: Some comments might make this chunk of code more understandable. The combination of switch expressions and inline delegates is difficult to grok at first.